uwh-refbox/src/main.rs

```diff
@@ -1497,9 +1703,19 @@ where
 {
     let min = secs / T::from(60u8);
     let sec = secs % T::from(60u8);
-    format!("{:2}:{:02}", min, sec)
+    format!("{:>2}:{:>02}", min, sec)
```

Comment:
Just curious, why this change? Right justify should be the default anyway

Reply:
oh, I thought left justify was the default